### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KiranHRubix**
updated rubixgoplatform build for mac to include both arm64 and amd64 builds

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,10 +10,16 @@ compile-windows:
 	go build -o windows/rubixgoplatform.exe
 
 compile-mac:
-	echo "Compiling for MacOS"
+	echo "Compiling for MacOS arm64"
```
Comment (vyshnavsdeepak) [inline Makefile:13]: Similar message down too

File: Makefile [hunk 2]
```diff
@@ -10,10 +10,16 @@ compile-windows:
...
 	go env -w GOARCH=arm64
 	go env -w CGO_ENABLED=1
-	go build -o mac/rubixgoplatform
+	go build -o mac/rubixgoplatform_arm64
+	go env -w GOARCH=amd64
+	go env -w CGO_ENABLED=1
+	go build -o mac/rubixgoplatform_amd64
+
+clean:
+	rm -f linux/rubixgoplatform windows/rubixgoplatform.exe mac/rubixgoplatform_arm64 mac/rubixgoplatform_amd64
```
Comment (vyshnavsdeepak) [inline Makefile:24]: deleting the entire folder will be better, right? That will go a long way when we build or download more files to the build folder.

rm -f linux windows mac

Comment (KiranHRubix) [reply Makefile]: done

**Comment by vyshnavsdeepak** [comment]
I see you have not created a branch out of develop but directly committed to develop.
While it works for now, your fork's develop branch's history and the destination's history won't match after the merge.

You'll find an issue from the next PR you create.

**Comment by vyshnavsdeepak** [comment]
PR is just titled as develop, change it to something relevant while merging.

**Comment by vyshnavsdeepak** [comment]
https://github.com/rubixchain/rubixgoplatform/issues/8#issue-1553048215

build folders - linux, windows and mac should be added to gitgnore

**Pull request closed without merging**